tailwind.config.ts

```diff
@@ -1,17 +1,19 @@
-import type { Config } from 'tailwindcss';
-
-export default {
-  content: ['./src/**/*.{js,ts,jsx,tsx,md,mdx}'],
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  darkMode: 'class', // Enable dark mode using the 'class' strategy
+  content: [
+    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
+    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
```

Comment:
also just curious if there is a specific idea behind adding only few directories ?

@Kishan1835